utils/client.ts

```diff
@@ -49,10 +53,23 @@ export const wagmi = createConfig({
   connectors: [
     ...(isE2E && e2e
       ? [new E2EConnector({ chains, ...e2e })]
-      : [...w3mConnectors({ projectId: walletConnectId, chains }), new SafeConnector({ chains })]),
+      : [
+          new InjectedConnector({ chains, options: { shimDisconnect: true } }),
+          new WalletConnectConnector({ chains, options: { projectId: walletConnectId, showQrModal: false } }),
+          new SafeConnector({ chains }),
+        ]),
```

Comment:
add [coinbase wallet](https://wagmi.sh/react/connectors/coinbaseWallet) (as the second option) and [ledger](https://wagmi.sh/react/connectors/ledger) connectors.